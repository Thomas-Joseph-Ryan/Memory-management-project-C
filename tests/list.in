list keys
list entries
set a 1
list keys
set b 2
list keys
list entries
set A 3
list keys
list entries
list snapshots
snapshot
set c a
snapshot
list snapshots