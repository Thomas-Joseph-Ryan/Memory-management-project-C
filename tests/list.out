> no keys

> no entries

> ok

> a

> ok

> b
a

> b [2]
a [1]

> ok

> A
b
a

> A [3]
b [2]
a [1]

> no snapshots

> saved as snapshot 1

> ok

> saved as snapshot 2

> 2
1

> 
bye
